frontend/app/src/components/display/amount/AmountDisplay.vue

```diff
@@ -195,6 +196,8 @@ const isNaN = computed<boolean>(() => get(displayValue).isNaN());
 // Decimal place of `realValue`
 const decimalPlaces = computed<number>(() => get(displayValue).decimalPlaces() ?? 0);
 
+const zeroDisplay = computed(() => '0');
```

Comment:
Isn't a computed that always returns zero a bit redundant? a simple const should suffice

```suggestion
const zeroDisplay = '0';
```